internal/cmd/db_shell.go

```diff
@@ -174,7 +175,17 @@ var shellCmd = &cobra.Command{
 			return shell.RunShellLine(shellConfig, string(b))
 		}
 
-		return shell.RunShell(shellConfig)
+		err := shell.RunShell(shellConfig)
+		if err != nil {
+			shellErr := strings.Split(err.Error(), "\n")
+			var serverErr string
+			if 1 < len(shellErr) {
+				serverErr = shellErr[1]
+			}
```

Comment:
What is the reasoning behind this logic?
Note that if the error doesn't have more than one line, `serverErr` will be empty.
If you're trying to remove blank lines, you can use something like `strings.TrimSpace`.

Reply:
Thanks for the review! The reasoning here was to only print the string that was added in this PR: https://github.com/tursodatabase/libsql-client-go/pull/81/files
(A string that displays the specific server connection error)

I would love to work on it, but I would like to know: in a perfect world, what would the error message the user received look like when being unable to access the shell for any reason?

I didn't see any documentation about error message standards (besides the use of internal.Emph()), so let me know if there is something I should know.

Thanks again :)